services/app-api/prisma/schema.prisma

```diff
@@ -105,41 +167,129 @@ model UpdateInfoTable {
-  unlockedRates RateRevisionTable[] @relation("unlockRateInfo")
-  submitedRates RateRevisionTable[] @relation("submitRateInfo")
+  unlockedRates     RateRevisionTable[]     @relation("unlockRateInfo")
+  submitedRates     RateRevisionTable[]     @relation("submitRateInfo")
+}
+
+model SharedRateCertifications {
+  id             String       @id @default(uuid())
+  createdAt      DateTime     @default(now())
+  updatedAt      DateTime     @default(now()) @updatedAt
+  contract       ContractRevisionTable[]
+  rateRevision   RateRevisionTable[]
+}
+
+model ActuaryContact {
+  id                 String            @id @default(uuid())
+  createdAt          DateTime          @default(now())
+  updatedAt          DateTime          @default(now()) @updatedAt
+  name               String
+  titleRole          String
+  email              String
+  actuarialFirm      ActuarialFirm
+  actuarialFirmOther String?
+
+  contractRevisionID String?
+  contractRevision   ContractRevisionTable? @relation(fields: [contractRevisionID], references: [id])
+
+  rateRevisionID     String?
+  rateRevision       RateRevisionTable?     @relation(fields: [rateRevisionID], references: [id])
+}
+
+model ContractDocument {
+  id                 String             @id @default(uuid())
+  createdAt          DateTime           @default(now())
+  updatedAt          DateTime           @default(now()) @updatedAt
+  name               String
+  s3URL              String
+  documentCategories DocumentCategory[]
+  sha256             String?
+  contractRevisionID String
+  contractRevision   ContractRevisionTable   @relation(fields: [contractRevisionID], references: [id])
+}
+
+model ContractSupportingDocument {
+  id                 String             @id @default(uuid())
+  createdAt          DateTime           @default(now())
+  updatedAt          DateTime           @default(now()) @updatedAt
+  name               String
+  s3URL              String
+  documentCategories DocumentCategory[]
+  sha256             String?
+  contractRevisionID String
+  contractRevision   ContractRevisionTable   @relation(fields: [contractRevisionID], references: [id])
+}
+
+model RateDocument {
+  id                 String             @id @default(uuid())
+  createdAt          DateTime           @default(now())
+  updatedAt          DateTime           @default(now()) @updatedAt
+  name               String
+  s3URL              String
+  documentCategories DocumentCategory[]
+  sha256             String?
+  rateRevisionID     String
+  rateRevision       RateRevisionTable       @relation(fields: [rateRevisionID], references: [id])
+}
+
+model RateSupportingDocument {
+  id                 String             @id @default(uuid())
+  createdAt          DateTime           @default(now())
+  updatedAt          DateTime           @default(now()) @updatedAt
+  name               String
+  s3URL              String
+  documentCategories DocumentCategory[]
```

Comment:
I think we can drop document categories now that we have separate Contract/Rate/Supporting document types